doc/api/process.md

```diff
@@ -1936,6 +1936,17 @@ added: v0.5.3
 
 A boolean value that is `true` if the current Node.js build includes support for IPv6.
 
+## `process.features.require_module`
```

Comment:
Should it be `require_esm`? Or is it purposefully vague so it includes also WASM module and whatever might come in the future? (should we specify that requiring CJS modules is supported on every version?)

Reply:
I think the ship has sailed after it's named `"module"` in package.json/loader hooks/`--experimental-require-module` - in Node.js API surface, `module` means "ESM", parallel to `wasm`.

Reply:
Wdyt of having a table (or list) of files that `require` is able to handle depending on if this value is true or false, so there's no way of misinterpreting what this mean?

Reply:
If we are documenting additional module types that can be loaded, then this value being true only means ESM can be loaded by `require()`. I am not sure what the table/list would contain other than one item, which has been documented here with a link - if we are also documenting what `require()` could already handle, that sounds like something that belongs to a different PR?

Reply:
Makes sense. Let's add a sentence like "The value here does not reflect the capacity of `require` to load other kind of modules, as documented in `modules.md`"

Reply:
I think you probably meant esm.md, but then in that document the only exception is wasm modules, and I am not sure who would see the documentation mentioning `loading ECMAScript modules` and think wasm modules, because even in esm.md it assumes "ECMAScript modules" means JavaScript modules written with the ECMAScript module syntax everywhere?

Reply:
No I mean `require()` can also load CJS module, even when `process.features.require_module === false`

Reply:
I see, you meant that "this value being false doesn't mean require() can't load modules that are not ECMAScript modules". I still feel that is a bit redundant though, because the description "A boolean value that is `true` if the current Node.js build supports loading ECMAScript modules using `require()`" already seems clear enough to me, and that basically repeats the same sentence with double negation.